rules-tattletale/addon/src/main/java/org/jboss/windup/rules/apps/tattletale/DisableTattletaleReportOption.java

```diff
@@ -0,0 +1,65 @@
+package org.jboss.windup.rules.apps.tattletale;
+
+import org.jboss.windup.config.AbstractConfigurationOption;
+import org.jboss.windup.config.InputType;
+import org.jboss.windup.config.ValidationResult;
+
+/**
+ *
+ * Indicates that the Tattletale report should NOT be generated.
+ *
+ * @author <a href="mailto:marcorizzi82@gmail.com">Marco Rizzi</a>
```

Comment:
Do we have to add our email in the classes we create ?

Reply:
I think it's good to have the author in a class.
I don't think we should consider mandatory to have the email there as well.